ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -102,9 +109,13 @@ if [[ -s "${SHARED_DIR}/xpn.json" ]]; then
   HOST_PROJECT_COMPUTE_SERVICE_ACCOUNT="$(jq -r '.computeServiceAccount' "${SHARED_DIR}/xpn.json")"
   HOST_PROJECT_CONTROL_SERVICE_ACCOUNT="$(jq -r '.controlServiceAccount' "${SHARED_DIR}/xpn.json")"
   HOST_PROJECT_PRIVATE_ZONE_NAME="$(jq -r '.privateZoneName' "${SHARED_DIR}/xpn.json")"
-else
-  # Set HOST_PROJECT to the cluster project so commands with `--project` work in both scenarios.
```

Comment:
Though I know when we did not append --project option for gcloud command, it will use the default login service project, but to respect the original logic, I still suggest to keep this logic.

Reply:
So init PROJECT_OPTION with "--project ${PROJECT_NAME}"?

Reply:
Maybe we can try this:
```
PROJECT_OPTION=""
if [[ -s "${SHARED_DIR}/xpn.json" ]]; then
  ...
  if [[ -n "${GOOGLE_CLOUD_XPN_SA}" ]]; then
    echo "Using QE XPN configurations..."
    PROJECT_OPTION="--project ${HOST_PROJECT} --account ${GOOGLE_CLOUD_XPN_SA}"
  else
    PROJECT_OPTION="--project ${HOST_PROJECT}"
  fi
else
  # Set HOST_PROJECT to the cluster project so commands with `--project` work in both scenarios.
  PROJECT_OPTION="--project ${PROJECT_NAME}"
fi
```

Reply:
If so, the first line can be omitted, right?

Reply:
I think it is a good habit to always init a var, so I prefer to keep it.